### PR TITLE
Fix Adv Input Prb

### PR DESCRIPTION
**Pull request by pivilartisant**
changes :
- validation of amount field when click confirm fees 
- correct button preset behaviour when opening and closing popup modal 
- correct button behaviour when choosing between custom fees and preset fees
- custom confirmation redirect to send coins screen and not confirmation screen
- validation for "1ajhjha" custom fees

### CODE REVIEW

File: web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx [hunk 1]
```diff
@@ -52,12 +52,14 @@ function Modal({ ...props }) {
   function handlePresetGas() {
     setCustomFees(false);
     setPresetGasFees(true);
+    handleFees('1000');
```
Comment (Pierregvx) [inline web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx:55]: hmm so as soon as we select the radiobutton it overrides the fees ?

Comment (pivilartisant) [reply web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx:55]: yeah and sets it to standard fees

File: web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx [hunk 2]
```diff
@@ -52,12 +52,14 @@ function Modal({ ...props }) {
...
+    setErrorAdvanced(null);
   }
 
   function handleCustomGas() {
     setCustomFees(true);
     setPresetGasFees(false);
-    handleFees('');
+    handleFees('0');
```
Comment (Pierregvx) [inline web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx:62]: why do you  want to set it at 0 ? this will put 0 in the input field

Comment (pivilartisant) [reply web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx:62]: i changed it to '' :)

**Review by Thykof — APPROVED** [review]

**Review by mazmassa — APPROVED** [review]

**Pull request merged**